src/stacks_tx/neon_encoder.rs

```diff
@@ -629,6 +635,35 @@ impl NeonJsSerialize for TransactionSmartContract {
     }
 }
 
+impl NeonJsSerialize for TransactionTenureChange {
+    fn neon_js_serialize(
+        &self,
+        cx: &mut FunctionContext,
+        obj: &Handle<JsObject>,
+        _extra_ctx: &(),
+    ) -> NeonResult<()> {
+        let previous_tenure_end = cx.string(encode_hex(&self.previous_tenure_end));
+        obj.set(cx, "previous_tenure_end", previous_tenure_end)?;
+
+        let previous_tenure_blocks = cx.number(self.previous_tenure_blocks);
+        obj.set(cx, "previous_tenure_blocks", previous_tenure_blocks)?;
+
+        let cause = cx.number(self.cause as u8);
+        obj.set(cx, "cause", cause)?;
+
+        let pubkey_hash = cx.string(encode_hex(&self.pubkey_hash));
+        obj.set(cx, "pubkey_hash", pubkey_hash)?;
+
+        let signature = cx.string(encode_hex(&self.signature));
+        obj.set(cx, "signature", signature)?;
+
+        let signers = cx.string(encode_hex(&self.signers));
+        obj.set(cx, "signers", signers)?;
```

Comment:
Might be missing the dynamic `signers_len` here 🤔

Reply:
`signers_len` is just the length of the `signers` bitmap. Only an implementation detail related to the wire encoding format for variable length arrays.